pkg/apis/core/v1beta1/federatedtypeconfig_types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
```

Comment:
We should probably add a GoDoc comment suggesting the `StatusType` and `StatusCollection` fields are still in alpha.

Reply:
I don't think this needs to be handled in this PR, and I've filed #918 to track it and related work.